db/migrate/20161124170211_create_subnet_compute_ressources.rb

```diff
@@ -0,0 +1,14 @@
+class CreateSubnetComputeRessources < ActiveRecord::Migration
+  def up
+    create_table :subnet_compute_resources do |t|
+      t.references :compute_resource
```

Comment:
@ohadlevy thanks for taking time to look at this PR. I implemented it like it was already done for domains. All the code i've seen related to taxonomy was for rights management, orgs and locations, so i didn't take a look at it as a way to implement this. I didn't saw many documentation (developer handbook, blog, or videos/deepdive) about taxonomies. If you have some resources to share, i will be happy to take a look at it.